src/musubi_tuner/qwen_image_train_network.py

```diff
@@ -342,6 +347,7 @@ def load_transformer(
             accelerator.device,
             dit_path,
             attn_mode,
+            args.edit_version == "2511",
             split_attn,
```

Comment:
The argument order here seems to be wrong? 

Reference: https://github.com/kohya-ss/musubi-tuner/commit/cc03b043328240efbbc8fe8875517b582afe8a7a